demo/dev/index.htm

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="/checkout.js" data-paypal-checkout></script>
+    <script src="../../dist/checkout.js?debug=true" data-paypal-checkout></script>
```

Comment:
This was intentional, it's being served up by webpack-dev-server when you run `npm run dev`. Was that not working?

Reply:
It was not.

Reply:
No local overrides on this one.